client/src/pages/messaging/helpers/index.js

```diff
@@ -12,13 +12,10 @@ export {
 
 export const getUserSessionID = (channelID) => {
   const userID_storage = storage.get('session-user-uuid') || {};
-  const userId = userID_storage.channelID === channelID ? userID_storage.userId : null;
-  return userId;
+  return userID_storage.channelID === channelID ? userID_storage.userId : null;
```

Comment:
```suggestion
const userId = userID_storage.channelID === channelID ? userID_storage.userId : null;
return userId;
```
I'll prefer to keep it like before for better readability.